### PR TITLE
Revise code of conduct and add community moderators policy

### PR DESCRIPTION
**Pull request by barbaricyawps**
Relates to: [website-hugo 21](https://github.com/thegooddocsproject/website-hugo/issues/21)

## Purpose / why

This PR adds a newly revised code of conduct as well as the community moderators policy document.

## What changes were made?

The existing code of conduct was replaced wholesale. This code of conduct was drafted by the GDP code of conduct committee and approved by the GDP community.

## Verification

The markdown version of these documents should look similar to their draft versions in Google Docs:

- [Good Docs Code of Conduct - DRAFT](https://docs.google.com/document/d/1tsiOyBPKRlEK3w_xVUMC0ipT8GLRMI8UVIavJOj9KH4/edit?usp=sharing)
- [Community Moderators - DRAFT](https://docs.google.com/document/d/1Kf9WXfmpU8qHys-3_WgBUEKEGIsXpSUAa1fymCoDSus/edit?usp=sharing)

---

## Checklist

> __Pull-request reviewer__ should ensure the following

* [ ] Are issues linked correctly?
* [ ] Is this PR labeled correctly?
* [ ] If template updates: do they align with [developers.google.com/style/](https://developers.google.com/style/)?
* [ ] Did the PR receive at least one :+1: and no :-1: from core-maintainers?
* [ ] On merging, did you complete the merge using [keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#manually-linking-a-pull-request-to-an-issue)?
* [ ] On merging, did you add any applicable notes to a [draft release](https://github.com/thegooddocsproject/templates/releases) and link to this PR?

### CODE REVIEW

File: Community_moderators.md [hunk 1]
```diff
@@ -0,0 +1,472 @@
+# Community Moderators - The Good Docs Project
```
Comment (ddbeck) [inline Community_moderators.md]: minor suggestion: this file could be named like the others in this directory (either all caps, `COMMUNITY_MODERATORS.md`, or Pascal case, `CommunityModerators.md`)

**Review by ddbeck — DISMISSED** [review]
As one of the CoC team members, I approve and confirm that this closely resembles the Google Doc that came out of that process. 👍 I did make one tiny suggestion for the community moderators filename, in a line comment, but that's not a blocker. Thank you!

File: Community_moderators.md (hunk 1 above)
Comment (barbaricyawps) [reply Community_moderators.md]: I made the change on my local copy, but for some reason it's not fully changing it in my commit. I'll see if I can fix this!

**Review by bwklein — APPROVED** [review]
This all looks good to me.  Once merged I will use them on the new website project.

**Pull request merged**